packages/wdio-browserstack-service/src/util.ts

```diff
@@ -0,0 +1,47 @@
+import { BROWSER_DESCRIPTION } from './constants'
+
+type Capabilities = WebDriver.Capabilities & WebdriverIO.MultiRemoteCapabilities;
+
+type Browser = WebdriverIO.BrowserObject & WebdriverIO.MultiRemoteBrowserObject;
```

Comment:
we have defined these in `service.ts` already. I suggest to move those into a `types.ts` and import them here.